lib/config.js

```diff
@@ -46,7 +46,7 @@ module.exports.getAssetsDir = () => mergedConfig.assetsDir;
 module.exports.getStaticDir = () => (mergedConfig.static === true ? mergedConfig.staticDir : mergedConfig.static);
 module.exports.getHost = () => mergedConfig.host;
 module.exports.getPort = () => mergedConfig.port;
-module.exports.getWatch = () => Boolean(mergedConfig.watch);
+module.exports.getWatch = () => Boolean(mergedConfig.watch || mergedConfig.w);
```

Comment:
Thanks for the PR, @HoverBaum! However, I think this should be fixed somewhere here: https://github.com/webpro/reveal-md/blob/master/bin/reveal-md.js#L13-L22

Reply:
That sounds reasonable. The library sued should probably handle setting `watch` for both `-w` and `--watch`.

Will tackle this angle tomorrow.